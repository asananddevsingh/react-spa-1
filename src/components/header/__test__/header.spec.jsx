import React from "react";
import { configure, shallow } from "enzyme";
import Header from "../header";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("Grocery component", () => {
  const wrapper = shallow(<Header />);
  it("should generate and match snapshot", async () => {
    expect(wrapper).toMatchSnapshot();
  });
});
