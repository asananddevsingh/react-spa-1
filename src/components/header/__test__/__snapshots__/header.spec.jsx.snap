// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grocery component should generate and match snapshot 1`] = `ShallowWrapper {}`;
